packages/ui/Checkbox/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
better add these to package.json as peerDependency

Reply:
and there's no need to add the `import React` line in order to use jsx after react 17 https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports

Reply:
1. Yes, and may need to add more when the ui design is confirmed
2. Agree. How about @cfxjs/component-message?

Reply:
Maybe `import React` is needed in Jest, from @RickStormBringer

Reply:
Yes, that is a good question and i suffer it a lot.Babel does not support  some features of React 17. So i manual add "import React" in every file.